discovery-provider/src/challenges/challenges.json

```diff
@@ -12,5 +12,11 @@
 		"amount": "5",
 		"active": false,
 		"step_count": 7
+	},
+	{
+		"id": "connect-verified",
+		"type": "boolean",
+		"amount": "5",
```

Comment:
Not sure where to check for this amount.

Reply:
From the figma doc, looks like this should be 10. Not sure if those are confirmed amounts though
https://www.figma.com/file/4M7qsTEjsbfJx66899ouKi/Solana-Rewards?node-id=101%3A141